ThreeBotPackages/tfgrid_solutions/kubernetes_cluster/chatflows/kubernetes_cluster_deploy.py

```diff
@@ -0,0 +1,83 @@
+from Jumpscale import j
+import netaddr
+import ipaddress
+
+
+def chat(bot):
+    """
+    """
+    user_info = bot.user_info()
+    name = user_info["username"]
+    email = user_info["email"]
+    ips = ["IPV6", "IPV4"]
+    default_cluster_name = name.split(".3bot")[0]
+
+    explorer = j.clients.threebot.explorer
+    if not email:
+        raise j.exceptions.BadRequest("Email shouldn't be empty")
+
+    ip_version = bot.single_choice("Choose your ip version to be used", ips)
+
+    workers_number = bot.int_ask("Please add the number of worker nodes")  # minimum should be 1
```

Comment:
I think it would be nicer to put it:
"Please specify the number of worker nodes"